src/include/column.i

```diff
@@ -30,9 +30,11 @@ __col_insert_search_gt(WT_INSERT_HEAD *ins_head, uint64_t recno)
 	 */
 	ins = NULL;
 	for (i = WT_SKIP_MAXDEPTH - 1, insp = &ins_head->head[i]; i >= 0;)
-		if (*insp != NULL && recno >= WT_INSERT_RECNO(*insp)) {
-			ins = *insp;	/* GTE: keep going at this level */
-			insp = &(*insp)->next[i];
+		if ((local_ins = *insp) != NULL &&
+		    recno >= WT_INSERT_RECNO(*insp)) {
+			/* GTE: keep going at this level */
+			ins = local_ins;
+			insp = &local_ins->next[i];
```

Comment:
This doesn't look right to me, we're still reading `*insp` twice. I'd suggest getting rid of `local_ins`, I think `ins` is all you need (but it wouldn't hurt my feelings to have all functions use the same naming pattern).